pkg/components/velero/component.go

```diff
@@ -65,6 +65,10 @@ func newComponent() *component {
 		Namespace: "velero",
 		// Once we have more than one provider supported, we should remove the default value
 		Provider: "azure",
+		Metrics: &Metrics{
```

Comment:
Actually, just running render is fine here, because all defaults fields are false anyway, so we only ensure that the sub-struct is initialized for easy templating.